### ISSUE TITLE
[Idea :: Improvement] Add Convenience Property/Accessor for ureg 

### ISSUE THREAD

**Issue by danieldjewell** [hgrecco/pint-pandas#96]
I love pint and pint-pandas - one of the things I run into (and confused the heck out of me until I understood it) is doing things with pint DataFrames and having different UnitRegistries (usually because I've imported pint separately and forgot to go grab `pint_pandas.PintType.ureg`) 

What about adding a convenience / super simple way, when using pint_pandas, to get the unit registry object to help negate the "different UnitRegistry" errors? (This isn't 100% clear in the documentation and it isn't currently very intuitive, IMHO.)

My inspiration for a solution to this is the way [MetPy](https://github.com/Unidata/Metpy) integrates pint - see [this Metpy example](https://unidata.github.io/MetPy/latest/examples/calculations/Dewpoint_and_Mixing_Ratio.html#sphx-glr-examples-calculations-dewpoint-and-mixing-ratio-py).

Basically, the convention in MetPy is simple:
```python
from metpy.units import units as u
```

`u` is now the UnitRegistry that Metpy uses in all of its calculations, etc. (Metpy makes *extensive* use of pint Quantities as inputs for a bunch of calculation functions - see the [metpy.calc](https://github.com/Unidata/MetPy/blob/main/src/metpy/calc/basic.py) module for examples.)

Thoughts? 

I was thinking of maybe just adding an object called "units" to the base that would point to the UnitRegistry... so you could do something like:

```python
from pint_pandas import units as u
```

**Comment by danieldjewell** [comment]
I just mocked this up and it's literally adding one line to `__init__.py` and editing `__all__`...

**Comment by andrewgsavage** [comment]
There were a few issues opened due to different UnitRegistries problems before 'pint.get_application_registry()' was used. Haven't had any other issues opened since pint-pandas used it.

Could you give an example which shows where you've run into a problem?

**Comment by andrewgsavage** [comment]
closing since no response, reopen if needed.
